truss-chains/truss_chains/deployment/deployment_client.py

```diff
@@ -138,10 +137,8 @@ class _ChainSourceGenerator:
     def __init__(
         self,
         options: definitions.PushOptions,
-        gen_root: pathlib.Path,
```

Comment:
Small cleanup, this `gen_root` was passed around to allow callers to optionally pass in a given directory, but every callsite I tracked down defaulted to a generated temp dir